gradle/scripts/repositories.gradle

```diff
@@ -29,7 +29,11 @@ repositories {
   maven {
     url "https://linkedin.jfrog.io/artifactory/open-source/"
   }
-  mavenCentral()
+  maven {
+    url "https://linkedin.jfrog.io/artifactory/gobblin-hive/"
+  }
+  jcenter()
```

Comment:
Can we replace jcenter with maven central? We are trying to limit dependencies on jcenter because of its flakiness after becoming EoL.  This makes our CI flakey

See https://github.com/apache/gobblin/pull/3554 for similar type changes.

Reply:
Removed jcenter(). Will check if everything looks good from github CI side.